scripts/development/README.md

```diff
@@ -0,0 +1,62 @@
+
+### Introduction
+
+Hephaestus, also known as the v3 ImageBuilder, is a Golang service used to create container images. This document will guide you through the steps required to set up and run Hephaestus locally.
+
+### Prerequisites
+Before starting the setup process, ensure that you have access to the Hephaestus repository, Minikube installed on your system, and have set the environment variables as follows:
+`WEBHOOK_SERVER_CERT_DIR=/<your-path-to>/hephaestus/local-development/webhook-certs`
+
+### Setup Process
+In the Hephaestus repository, run go mod tidy to ensure that all dependencies are installed.
+Set the necessary webhook environment variable by adding the following program arguments: start -c local-development/hephaestus.yaml.
+Run the setup scripts available under the scripts/development directory. The scripts need to be run in the following order:
+1. `00-sanity-check.sh`: This script asserts that required executables are installed on the host.
+2. `01-setup-minikube.sh`: This script creates a Minikube cluster with adequate resources and an extended port range. Note that if you encounter the error "GUEST_DRIVER_MISMATCH," remove the --driver=hyperkit argument on line 16 in the Minikube start command.
+3. `02-install-helm-apps.sh`: This script installs required Kubernetes applications and CRDs such as RabbitMQ.
+4. `03-expose-buildkit.sh`: This script creates services that target individual BuildKit pods and maps their ingress hostnames so that they mimic the addresses generated from endpoints.
+5. `04-export-configurations.sh`: This script exports the controller configuration and webhook TLS files onto the host.
+6. Update your `local-development/hephaestus.yaml` file to the correct RabbitMQ port by running `minikube service rabbitmq` in a new terminal window.
+This command exposes the RabbitMQ service to your local machine and allows you to interact with it as if it were running on your local machine.
+After running this command, there will be a list towards the bottom of the command with available ports.
+```
+[default rabbitmq  http://127.0.0.1:50377
+http://127.0.0.1:53975
+http://127.0.0.1:53976
+http://127.0.0.1:53977]
+```
+Get the first port in the default service list _here it's 50377_,  update that port in two places. Examples in the code blocks below.
+- `local-development/hephaestus.yaml` on lines 36
+- `scripts/development/helmfile.yaml` on line 17.
+
+_local-development/hephaestus.yaml L36_
+```
+amqp:
+url: "amqp://user:roger-rabbit@localhost:50377"
+```
+
+_scripts/development/helmfile.yaml on L17_
+```
+releases:
+- name: rabbitmq
+  namespace: default
+  chart: bitnami/rabbitmq
+  version: 11.1.1
+  values:
+    - auth:
+      password: roger-rabbit
+      service:
+      type: NodePort
+      nodePorts:
+      amqp: 50377
+```
+
+7. In a separate terminal window, run `scripts/development/remote-context/expose.sh`
```

Comment:
@steved I wanted to add a little on what this script does for hephaestus, it looks like its compressing a docker file that installs Jupityr and serves it using an HTTP server implemented in Python. Do you know why this is needed for the manager to run?